lumigator/backend/backend/api/routes/jobs.py

```diff
@@ -78,7 +80,11 @@ def create_annotation_job(
         output_field="ground_truth",
     )
     inference_job_create_request.store_to_dataset = True
-    job_response = service.create_job(inference_job_create_request, background_tasks)
+    job_response = service.create_job(inference_job_create_request)
+
+    background_tasks.add_task(
+        service.handle_inference_job, job_response.id, inference_job_create_request
```

Comment:
The route package should not be responsible of starting up tasks, imho. Why should it be moved from the services package?

Reply:
The reasoning was because I'm trying to elevate job-specific logic out of the `create_job` function. I could make a small wrapper function in the service that creates the background task instead? Adding it here in the route seemed like the cleanest solution but I'm fine with moving the task startup into the service layer if you'd like?

Reply:
If you don't mind, yes, I'd prefer it. In that way the routes would handle only adaptation to services, and schema changes, but not other mechanisms like job execution or scheduling. Thanks a lot ^_^